### ISSUE TITLE
gold-zip-input accessibility issues

### ISSUE THREAD

**Issue by lpalmaro** [PolymerElements/gold-zip-input#5]
Audit results: 
[Severe] Controls and media elements should have labels (6)
[Warning] Text elements should have a reasonable contrast ratio (3)
Not applicable tests (9)
Passing tests (6)

Contrast: the text label for Zip Code is too light 

Firefox / NVDA: text entry fields speak "has autocomplete"

**Comment by notwaldorf** [comment]
Done! :tada:
